components/raftstore/src/store/fsm/store.rs

```diff
@@ -828,28 +799,69 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport> PollHandler<PeerFsm<EK, ER>, St
     }
 
     fn end(&mut self, peers: &mut [Box<PeerFsm<EK, ER>>]) {
-        self.flush_ticks();
+        let now = TiInstant::now();
```

Comment:
If these is put to the end of the function, the flush of metrics may be reduce by at lease one time. Otherwise L856 will always mutate the metrics.